circleci/images/exttester/Dockerfile

```diff
@@ -34,6 +34,16 @@ WORKDIR /build/
 RUN curl -fLO "http://ftp.postgresql.org/pub/source/v${PG_VERSION}/postgresql-${PG_VERSION}.tar.bz2"
 
 RUN tar jxf "postgresql-${PG_VERSION}.tar.bz2"
+
+# apply optional patches that might be required for a successful testsuite
+WORKDIR /build/postgresql-${PG_VERSION}/
+COPY patches/ patches/
+SHELL ["/bin/bash", "-c"]
+RUN if [ -d "patches/${PG_VERSION}/" ]; \
```

Comment:
I like this idea, it is just sad that one mistake in postgres side causes many others to resolve the problem multiple times.

Reply:
Only half of the year ;)